README.md

```diff
@@ -48,18 +49,34 @@ All the available codemods to update your express server:
 
 <!-- CODEMODS START -->
 
-### magic redirect (5.0.0)
+### magic redirect (v5.0.0)
 
 Transform the deprecated magic string "back"
 
-### pluralized methods (5.0.0)
+### pluralized methods (v5.0.0)
 
 Transform the methods to their pluralized versions
 
-### v4 deprecated signatures (5.0.0)
+### v4 deprecated signatures (v5.0.0)
 
 Transform the deprecated signatures in Express v4
 
+### redirect (v5.0.0)
+
+Reverse param order for "redirect" method
+
+### req param (v5.0.0)
+
+Change request.param() to dedicated methods
+
+### send file (v5.0.0)
```

Comment:
This could also go in v4 as deprecated signatures, I think.

Reply:
right now it's a separate codemod. We can merge this all into one codemod in next PR